### PR TITLE
Added SetBit.js, SetBit.test.js in Bit-Manipulation directory

### PR DESCRIPTION
**Pull request by AbhinavXT**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new JavaScript files are placed inside an existing directory.
* [X] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [X] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Bit-Manipulation/test/SetBit.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { setBit } from '../SetBit'
+
+test('should set bit at the given bit Position', () => {
```
Comment (raklaptudirm) [inline Bit-Manipulation/test/SetBit.test.js:3]: Expand the test message a bit to give as much information as possible.
Here:
```suggestion
test('Set bit number 0 in 1:', () => {
```

Comment (raklaptudirm) [reply Bit-Manipulation/test/SetBit.test.js:3]: Add this for all the tests.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

**Comment by raklaptudirm** [comment]
#630 Progress.

File: Bit-Manipulation/test/SetBit.test.js (hunk 1 above)
Comment (raklaptudirm) [reply Bit-Manipulation/test/SetBit.test.js]: @AbhinavXT Do not apply the same name to all of them, replace the 0 and 1 with the bit number and the number respectively.

Comment (AbhinavXT) [reply Bit-Manipulation/test/SetBit.test.js]: Yes, Sorry about that. I will correct the changes.

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**